content/docs/getting_started/upgrade.md

```diff
@@ -51,7 +51,7 @@ For OSM versions `x.y.z` where `x >= 1`:
 The recommended way to upgrade a mesh is with the `osm` CLI. For advanced use cases, `helm` may be used.
 
 ### CRD Upgrades
-Because Helm does not manage CRDs beyond the initial installation, OSM leverages a helm hook to update existing and new CRDs during an upgrade. If the new release contains updates to existing CRDs or adds new CRDs, the [crd-upgrade-hook](https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/crds-upgrade-hook.yaml) will update the CRDs. The associated Custom Resources will remain as is, requiring no additional action prior or immediately after the upgrade.
+Because Helm does not manage CRDs beyond the initial installation, OSM leverages an init-container on `osm-bootstrap` pod to to update existing and add new CRDs during an upgrade. If the new release contains updates to existing CRDs or adds new CRDs, the [init-osm-bootstrap](https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/osm-bootstrap-deployment.yaml#L34) on the `osm-bootstrap` pod will update the CRDs. The associated Custom Resources will remain as is, requiring no additional action prior or immediately after the upgrade.
```

Comment:
Remove ref so we don't need to update it during a release.

```suggestion
Because Helm does not manage CRDs beyond the initial installation, OSM leverages an init-container on the `osm-bootstrap` pod to update existing and add new CRDs during an upgrade. If the new release contains updates to existing CRDs or adds new CRDs, the `init-osm-bootstrap` on the `osm-bootstrap` pod will update the CRDs. The associated Custom Resources will remain as is, requiring no additional action prior to or immediately after the upgrade.
```